R/create_IV.R

```diff
@@ -53,17 +56,30 @@ create_IV <- function(data,
     train <-
       data %>%
       rename(outcome = outcome) %>%
-      select(predictors, outcome) %>%
+      select(all_of(predictors), outcome) %>%
```

Comment:
I would add `tidyselect::all_of()` here to make the NAMESPACE explicit - to avoid warnings / errors in checks.